Assets/MixedRealityToolkit.SDK/Experimental/Features/UX/BoundsControl/Visuals/BoundsControlVisualUtils.cs

```diff
@@ -0,0 +1,109 @@
+﻿using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.UI.Experimental.BoundsControlTypes;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI.Experimental
+{
+    /// <summary>
+    /// Helper class providing some static utility functions for <see cref="BoundsControl"/> <see cref="BoundsControlHandlesBase">handles</see>/>
+    /// </summary>
+    internal class BoundsControlVisualUtils
```

Comment:
Nit: Not a fan of `internal` methods - classes even less so. Would like to hear justification if you'll indulge me.

Reply:
i don't want anyone else to access those as they're purely for internal bounding box or now bounds control use. there's a lot of methods i had to make public so bounding box could access them but there's no reason to access them from anywhere else. if i make them public we'd have to support all of them / changing them would introduce a breaking change etc. 
In c++ i would've made all of them private and friended boundscontrol. afais making them internal is the closest I can do to that in c#